composer.json

```diff
@@ -17,8 +17,8 @@
     },
     "require": {
         "php": "^5.6 || ^7.0",
-        "container-interop/container-interop": "^1.1",
         "psr/http-message": "^1.0",
+        "psr/container": "^1.0",
```

Comment:
We can make this fully BC by requiring:

- `"container-interop/container-interop": "^1.2"`
- `"psr/container": "^1.0"`

That way, if folks are still type-hinting on container-interop interfaces, their code will continue to work. We can message then that we'll be removing support for container-interop in the next major version (v3).

One reason we can't go full-on with psr/container quite yet is because none of the supported containers currently support it. Until there are versions that do, we'd actually make Expressive useless!